echo_handlers.go

```diff
@@ -459,5 +349,22 @@ func (svc *Service) LogoutHandler(c echo.Context) error {
 		sess.Options.Domain = svc.cfg.CookieDomain
 	}
 	sess.Save(c.Request(), c.Response())
-	return c.Redirect(302, "/")
+	return c.NoContent(http.StatusNoContent)
+}
+
+func (svc *Service) InfoHandler(c echo.Context) error {
+	csrf, _ := c.Get(middleware.DefaultCSRFConfig.ContextKey).(string)
+	user, err := svc.GetUser(c)
+	if err != nil {
+		return err
```

Comment:
we need to fix all these places where we directly return the `err`. It doesn't seem to be a valid response